src/summarycode/summarizer.py

```diff
@@ -112,7 +112,7 @@ def summarize_codebase(codebase, keep_details, **kwargs):
         ('holders', holder_summarizer),
```

Comment:
Let's avoid most changes in summarycode if possible: conceptually the summaries are for packages, not for manifests